src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * List Widget
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ * @since   TBD
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Widget\Widget_Abstract;
+
+/**
+ * Class for List Widget.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class Widget_List extends Widget_Abstract {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $slug = 'tribe_events_list_widget';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $view_slug = 'widget-list';
```

Comment:
:no_entry_sign: Missing @var tag in member variable comment

<sub>hash: 503177b51e39585d35b2ea3e2c90d1f9</sub>